compiler/src/iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenDialect.cpp

```diff
@@ -52,41 +52,177 @@ void IREECodegenDialect::initialize() {
   addTypes<IREE::Codegen::NullPointerType>();
 }
 
+static LogicalResult validateTuningEntrypoint(
+    ModuleOp moduleOp, transform::NamedSequenceOp &kernelConfigOp,
+    int &numTuningEntryPoints, const std::string &kRequiredByDefaultAttr) {
```

Comment:
Only use the `k` prefix for compile-time constants. Also, you can pass this as StringRef